server/models/Request.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const requestSchema = new mongoose.Schema({
+    user: {
+        type: mongoose.Schema.Types.ObjectId,
+   		ref: 'profile'
+    },
+    sitter: {
+        type: mongoose.Schema.Types.ObjectId,
+    	ref: 'profile'
+    },
```

Comment:
Spacing is also strange here, if you use VS Code, you can set Prettier to run automatically whenever you save